lighthouse-core/audits/dobetterweb/geolocation-on-start.js

```diff
@@ -40,15 +48,26 @@ class GeolocationOnStart extends Audit {
     if (typeof artifacts.GeolocationOnStart === 'undefined' ||
         artifacts.GeolocationOnStart === -1) {
       return GeolocationOnStart.generateAuditResult({
-        rawValue: false,
-        debugString: 'Unable to get geolocation values.'
+        rawValue: -1,
+        debugString: 'GeolocationOnStart gatherer did not run'
```

Comment:
I'm treating this like the rest of the collectUsage audits. `artifacts.GeolocationOnStart === -1` is the error case where the gatherer promise rejects